wikidatarefisland/data_access/wdqs_reader.py

```diff
@@ -0,0 +1,35 @@
+from SPARQLWrapper import JSON, SPARQLWrapper
+
+
+class WdqsReader(object):
+    def __init__(self, user_agent, url):
+        self.user_agent = user_agent
+        self.url = url
+
+    def get_sparql_data(self, query):
+        sparql = SPARQLWrapper(self.url, agent=self.user_agent)
+        sparql.setQuery(query)
+        sparql.setReturnFormat(JSON)
+        return sparql.query().convert()['results']['bindings']
+
+    def get_usecases(self, pid):
+        query = """SELECT ?item ?value
+WHERE {
+?item wdt:""" + pid + """ ?value
+}
+
+LIMIT 10
+        """
+        return self.get_sparql_data(query)
+
+    def get_schemaorg_mapping(self):
```

Comment:
IMHO I think we should also test these methods, as they clearly will be used in the pipeline and services.

Reply:
yeah, I was thinking about it, it has quite amount of business logic in it but I don't know how that can be tested, unless I make it full integration test and test it against production.

Reply:
I wrote a test that checks against the query directly, that's the most we can do I think :/